test/guardian/permissions/bitwise_test.exs

```diff
@@ -0,0 +1,298 @@
+defmodule Guardian.Permissions.BitwiseTest do
+  use ExUnit.Case, async: true
+
+  import Plug.Test
+
+  alias Guardian.Plug, as: GPlug
+  alias GPlug.Pipeline
+  alias Guardian.Permissions.Bitwise, as: GBits
+
+  defmodule Impl do
+    use Guardian, otp_app: :guardian,
+                  permissions: %{
+                    user: [:read, :write],
+                    profile: %{read: 0b1, write: 0b10}
+                  },
+                  token_module: Guardian.Support.TokenModule
+
+    use Guardian.Permissions.Bitwise
+
+    def subject_for_token(resource, _claims), do: {:ok, resource}
+    def resource_from_claims(claims), do: {:ok, claims["sub"]}
+
+    def build_claims(claims, _resource, opts) do
+      claims
+      |> encode_permissions_into_claims!(Keyword.get(opts, :permissions))
```

Comment:
We don't need to use pipes for single values or functions: https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines